apps/menu-bar/index.js

```diff
@@ -2,11 +2,13 @@
  * @format
  */
 
-import { AppRegistry } from 'react-native';
+import registerRootComponent from 'expo/build/launch/registerRootComponent';
 import 'react-native-url-polyfill/auto';
 
-import { name as appName } from './app.json';
 import App from './src/App';
 import './src/windows';
 
-AppRegistry.registerComponent(appName, () => App);
+// registerRootComponent calls AppRegistry.registerComponent('main', () => App);
+// It also ensures that whether you load the app in the Expo client or in a native build,
+// the environment is set up appropriately
+registerRootComponent(App);
```

Comment:
We need to use expo `registerRootComponent` for react-native-web to work out of the box